tecken/base/symboldownloader.py

```diff
@@ -200,15 +193,9 @@ def invalidate_cache(self, symbol, debugid, filename):
                 # exists_in_source(source, key)
                 # But that function is wrapped and now has an extra
                 # function to "undoing" it.
-                if source.is_google_cloud_storage:
-                    bucket = source.get_or_load_bucket()
-                    exists_in_source.invalidate(
-                        bucket, self._make_key(prefix, symbol, debugid, filename)
-                    )
-                else:
-                    exists_in_source.invalidate(
-                        source, self._make_key(prefix, symbol, debugid, filename)
-                    )
+                exists_in_source.invalidate(
+                    source, self._make_key(prefix, symbol, debugid, filename)
+                )
```

Comment:
This is typical of the application changes. A ``if`` statement is used for the GCS code path, and a ``else`` for the S3 path. Removing GCS gets rid of the ``if`` statement and de-dents the code.

Reply:
:+1: